src/Microsoft.Identity.Web.TokenCache/Microsoft.Identity.Web.TokenCache.csproj

```diff
@@ -82,14 +82,14 @@
     <DocumentationFile>Microsoft.Identity.Web.xml</DocumentationFile>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net472' Or '$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'net5.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'net5.0'">
     <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="5.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="5.0.8" />
     <PackageReference Include="System.Text.Encodings.Web" Version="5.0.1" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' or '$(TargetFramework)' == 'net472'">
```

Comment:
@jennyf19 @jmprieur @pmaytak - this package has no conditional compilation at all, so the only reason we have all these frameworks it to be able to have different dependencies, i.e. different versions of dependencies.

In particular, net462 and net472 seem identical?

If we get net4*2 to have the same dependencies as netstandard, we might as well eliminate net4*2 completely.

Reply:
I don't remember the details of the investigation into the target framework, but I see that most packages try to simplify things into supporting only `netstandard2.0`.

Reply:
I will note that while I was trying to assemble a simple repro of this issue, a project on net462 used the netstandard dependency set for some reason, while net472 used the expected net472 dependencies. I didn't look into it too deeply though, my org's projects are all net472.